scrapyd_k8s/api.py

```diff
@@ -11,6 +12,16 @@
 config = Config()
 repository = (config.repository_cls())(config)
 launcher = (config.launcher_cls())(config)
+scrapyd_config = config.scrapyd()
+
+config_username = scrapyd_config.get("username")
+config_password = scrapyd_config.get("password")
+
+if not (config_username is None or config_password is None):
```

Comment:
perhaps rewrite as `if config_username is not None and config_password is not None` for better readability